types/redact-secrets/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for redact-secrets 1.0
+// Project: https://github.com/watson/redact-secrets#readme
+// Definitions by: wrumsby <https://github.com/wrumsby>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
```

Comment:
Note that versions prior to 3.0 are no longer supported on DefinitelyTyped, so I wouldn't expect this to have any effect.